packages/oui-checkbox/README.md

```diff
@@ -44,38 +44,32 @@
+  <oui-checkbox description="Lorem ipsum dolor sit amet, consectetur adipiscing elit.">Normal</oui-checkbox>
+  <oui-checkbox description="Pellentesque euismod magna rutrum lectus gravida semper." disabled>Disabled</oui-checkbox>
 </div>
 ```
 
 ### On change
 
-**Note:** Model will not be refreshed until the `on-change` callback as not finished. If you want to access the new model inside the `on-change` callback you need to use the `modelValue` variable as below.
+**Note**: Model will not be refreshed until the `on-change` callback as not finished. If you want to access the new model inside the `on-change` callback you need to use the `modelValue` variable as below.
 
 ```html:preview
 <oui-checkbox
-  text="Normal"
   model="$ctrl.onChangeValue"
   on-change="$ctrl.lastOnChangeValue = modelValue"
-></oui-checkbox>
+>Normal</oui-checkbox>
 
 <span>Last onChange value: {{ $ctrl.lastOnChangeValue }}</span>
 ```
 
-### Accessibility
-
-Needs to be done.
-
 ## API
 
-| Attribute     | Type                    | Binding | One-time Binding | Values                   | Default | Description
-| ----          | ----                    | ----    | ----             | ----                     | ----    | ----
-| text          | string                  | @       |                  |                          |         | checkbox text
-| description   | string                  | @?      |                  |                          |         | description text
-| id            | string                  | @?      | `true`           |                          |         | id attribute of the checkbox
-| name          | string                  | @?      | `true`           |                          |         | name attribute of the checkbox
-| disabled      | boolean                 | <?      |                  |                          | false   | disabled flag
-| model         | nullable&lt;boolean&gt; | =?      |                  | `true`, `false`, `null`  |         | current value of the checkbox and null is considered as `indeterminate`
-| required      | boolean                 | <?      |                  |                          | false   | `true` if the checkbox should be checked
-| on-change     | function                | &?      |                  |                          |         | handler triggered when value has changed
+| Attribute     | Type                    | Binding | One-time Binding  | Values                   | Default    | Description
+| ----          | ----                    | ----    | ----              | ----                     | ----       | ----
+| `model`       | nullable&lt;boolean&gt; | =?      | no                | `true`, `false`, `null`  | n/a        | current value of the checkbox and null is considered as `indeterminate`
```

Comment:
`nullable<boolean>` -> `boolean?` to replicate C# behavior https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/nullable-types/?

or

`nullable<boolean>` -> `boolean | null`to replicate TypeScript behaviors https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html ?